src/main/java/students/student_leonards_petkevics/lesson_2/task1.java

```diff
@@ -0,0 +1,38 @@
+package main.java.students.student_leonards_petkevics.lesson_2;
+
+import java.util.Scanner;
+
+public class task1 {
+    public static void main(String[] args){
+
+
+        int firstnumber;
```

Comment:
```suggestion
        int firstNumber;
```
В названиях переменных мы использует стиль camelCase